examples/controlflow/docker-compose.yaml

```diff
@@ -1,4 +1,3 @@
-version: "3.7"
```

Comment:
```
WARN[0000] /Users/nate/github.com/humanlayer/humanlayer/examples/controlflow/docker-compose.yaml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion
```

Reply:
cool